tests/test_csp.py

```diff
@@ -2,6 +2,52 @@
 from csp import *   #noqa
 
 
+def test_csp_assign():
+    var = 10
+    val = 5
+    assignment = {}
+    australia.assign(var, val, assignment)
+
+    assert australia.nassigns == 1
+    assert assignment[var] == val
+
+
+def test_csp_unassign():
+    var = 10
+    assignment = {var: 5}
+    australia.unassign(var, assignment)
+
+    assert (var in assignment) == False
+
+
+def test_csp_nconflits():
+    map_coloring_test = MapColoringCSP(list('RGB'), 'A: B C; B: C; C: ')
+    assignment = {'A': 'R', 'B': 'G'}
+    var = 'C'
+    val = 'R'
+    assert map_coloring_test.nconflicts(var, val, assignment) == 1
+
+    val = 'B'
+    assert map_coloring_test.nconflicts(var, val, assignment) == 0
+
+
+def test_csp_actions():
+    map_coloring_test = MapColoringCSP(list('123'), 'A: B C; B: C; C: ')
+
+    state = {'A': '1', 'B': '2', 'C': '3'}
+    assert map_coloring_test.actions(state) == []
+
+    state = {'A': '1', 'B': '3'}
+    assert map_coloring_test.actions(state) == [('C', '2')]
+
+    state = {'A': '1', 'C': '2'}
+    assert map_coloring_test.actions(state) == [('B', '3')]
+
+    state = {'A': '1'}
+    assert (map_coloring_test.actions(state) == [('C', '2'), ('C', '3')] or
+            map_coloring_test.actions(state) == [('B', '2'), ('B', '3')])
+
+
 def test_backtracking_search():
     assert (backtracking_search(australia) is not None) == True
```

Comment:
Instead of "assert (X is not None) == True", just do "assert X".
Here and at least one other place.